Home Page: Expand ARIA, fix link labels

mcking65 opened the pull request:
Fixes #2328 
Fixes #2378

mcking65 commented:
@richnoah @alflennik 

In 6d60ae8, I moved the link outside the paragraph, and now the link does not render.

alflennik commented:
Hi @mcking65! Only p tags are supported inside that section. Could you try wrapping the link in a p tag and see if that resolves it?

mcking65 commented:
@alflennik wrote:

> Hi @mcking65! Only p tags are supported inside that section. Could you try wrapping the link in a p tag and see if that resolves it?


Thank you Alex! That worked. I hope it looks OK visually.

mcking65 commented:
@jongund @charmarkk 

I finished the edits we discussed in the meeting. This is ready for review now.

I'd like to know if you think this fully addresses the feedback in #2327.

richnoah commented:
@mcking65 from a visual inspection all the requirements have been met.

charmarkk commented:
@mcking65 +1 to @richnoah  - LGTM.
